azkaban-common/src/test/java/azkaban/project/DirectoryFlowLoaderTest.java

```diff
@@ -65,4 +78,47 @@ public void testRecursiveLoadEmbeddedFlow() {
     // Should be 3 errors: jobe->innerFlow, innerFlow->jobe, innerFlow
     Assert.assertEquals(3, loader.getErrors().size());
   }
+
+  private static File decompressTarBZ2(InputStream is) throws IOException {
```

Comment:
Why do you choose to compress the project flows in .tar.bz2? Can we just use the existing .zip and the existing method to zip/unzip the project?

Reply:
the file is 51kb using BZ2 and 3MB using ZIP or tar/GZ.  I felt it was better to not put a 3mb ZIP file into git, so I went with BZ2.  I cant change it to ZIP if you prefer though.

Reply:
Ok, it makes sense to me.